.gitattributes

```diff
@@ -22,17 +22,17 @@
 *.ttf  binary
 
 # Ignore some meta files when creating an archive of this repository
+# We do not ignore any content, because this repo represents the 
+# `yiisoft/yii2-dev` package, which is expected to ship all tests and docs.
+/.appveyor.yml      export-ignore
```

Comment:
We aren't using AppVeyor anymore after switching to github actions.

Reply:
done.